packages/headless/rollup.config.js

```diff
@@ -186,15 +188,24 @@ function buildUmdOutput(outDir, name) {
     file: `${outDir}/headless.js`,
     format: 'umd',
     name,
-    sourcemap: isProduction
+    ...sourceMapConfig()
   }
 }
 
 function buildEsmOutput(outDir) {
   return {
     file: `${outDir}/headless.esm.js`,
     format: 'es',
+    ...sourceMapConfig(),
+  }
+}
+
+function sourceMapConfig() {
+  return {
     sourcemap: isProduction,
+    sourcemapPathTransform: (relativeSourcePath) => {
```

Comment:
I think that if this was used for the `node` output, this would break, right ?

Maybe we should then move this `dist/browser/` config at the `buildBrowserConfiguration` function level ?